test_cases/assert_violations/out-of-bounds-exception.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.4.19;
+
+contract OutOfBoundsException {
+
+	uint256[] public array;
+
+	function getArrayElement(uint256 idx) returns (uint256) {
+		return array[idx];
```

Comment:
This could throw.

Reply:
This is meant for an example that could throw but should *not* be reported as an assert violation. Does that make sense?

Reply:
Is this sample related to another variant around exception handling -> https://cwe.mitre.org/data/definitions/703.html ?

Reply:
Also it be great to have the definition SWC-110 completed :)

Reply:
Added in #48